### ISSUE TITLE
Rendering of "unilist" type is broken

### ISSUE THREAD

**Issue by oliwel** [openxpki/openxpki#889]
When using the new "request_info" type with a multivalued item, the list is not rendered very nicely
![grafik](https://github.com/openxpki/openxpki/assets/1262598/844c28f7-369c-4231-ae28-8a66d6d9c859)

Expected would be either a separator character or a "one item per line" view as with all other lists.

**Comment by oliwel** [comment]
```json
{
  "tooltip": "",
  "format": "unilist",
  "description": "Information about the Request.",
  "label": "Requester Information",
  "name": "cert_info",
  "value": [
    {
      "label": "Requester Name",
      "value": "Oliver Welter",
      "format": "text"
    },
    {
      "value": "owelter@whiterabbitsecurity.com",
      "format": "email",
      "label": "Email address"
    },
    {
      "label": "Company Name",
      "value": "WhiteRabbitSecurity",
      "format": "text"
    },
    {
      "value": "PKI",
      "format": "text",
      "label": "Department"
    },
    {
      "label": "Additional Email",
      "format": "email",
      "value": [
        "john@doe.com",
        "jane@doe.com"
      ]
    }
  ]
}
```

**Comment by maxhq** [comment]
Fixed in commit fea63e1fd
